LEAF_Request_Portal/js/form.js

```diff
@@ -93,10 +110,16 @@ var LeafForm = function(containerID) {
             }
             return conditionsLinkedToParent;
         }
-        const getConditionsLinkedToChild = (childID, currParentID)=> {
+        /**
+         *
+         * @param {number} childID id of a child condition
+         * @param {number} currParentID the id of the controller that was updated
+         * @returns array of all other parents that control the given child, or empty array
+         */
+        const getConditionsLinkedToChild = (childID=0, currParentID=0)=> {
```

Comment:
This was primarily to protect this from being undefined if there's an issue with the element's id.  I added an additional check just in case (and to skip the loops), but a 0 value should just result in an empty array, since there should not be conditions with parent or child indicator IDs of 0